definitions/variable/electricitysources_tags.yaml

```
@@ -0,0 +1,29 @@
+# This file adds needed variables for the EMPIRE data upload
+-  Secondary Energy|Electricity|{Electricity Source}:
+     description: Net electricity generation from {Electricity Source}
+     unit: EJ/yr
+- Investment|Energy Supply|Electricity|{Electricity Source}:
+    description: Investment in {Electricity Source}
+    unit: billion USD_2010/yr
+- Investment|Energy Supply|Electricity|{Common Electricity Source}:
+    description: Investment in {Common Electricity Source}
+    unit: billion USD_2010/yr
+- Variable Cost|Electricity|{Electricity Source}:
+    description: Variable cost for generating electricity from {Electricity Source} with the fuel cost
+    unit: EUR/MWh
+- Variable Cost|Electricity|{Common Electricity Source}:
+    description: Variable cost for generating electricity from {Common Electricity Source} with the fuel cost
+    unit: EUR/MWh
+- Capacity|Electricity|{Electricity Source}:
+    description: Installed (available) capacity of {Electricity Source}
+    unit: GW
+- Capital Cost|Electricity|{Electricity Source}:
+    description: Capital cost for {Electricity Source}
+    unit: USD_2010/kW
+- Emission Rate|CO2|Energy|Supply|Electricity|{Electricity Source}:
+    description: CO2 emission rate from {Electricity Source}
+    unit: tons/MWh
```

We usually include the emission-species in the units
```suggestion
    unit: t CO2/MWh
```